etc/input.conf

```diff
@@ -29,163 +29,152 @@
 # numeric value accordingly
-#WHEEL_UP      seek 10
-#WHEEL_DOWN    seek -10
-#WHEEL_LEFT    add volume -2
-#WHEEL_RIGHT   add volume 2
+#WHEEL_UP      seek 10          # seek 10 seconds forward
+#WHEEL_DOWN    seek -10         # seek 10 seconds backward
+#WHEEL_LEFT    add volume -2    # lower the volume
+#WHEEL_RIGHT   add volume 2     # raise the volume
 
 ## Seek units are in seconds, but note that these are limited by keyframes
-#RIGHT seek  5
-#LEFT  seek -5
-#UP    seek  60
-#DOWN  seek -60
+#RIGHT seek  5                          # seek 5 seconds forward
+#LEFT  seek -5                          # seek 5 seconds backward
+#UP    seek  60                         # seek 1 minute forward
+#DOWN  seek -60                         # seek 1 minute backward
 # Do smaller, always exact (non-keyframe-limited), seeks with shift.
 # Don't show them on the OSD (no-osd).
-#Shift+RIGHT no-osd seek  1 exact
-#Shift+LEFT  no-osd seek -1 exact
-#Shift+UP    no-osd seek  5 exact
-#Shift+DOWN  no-osd seek -5 exact
-# Skip to previous/next subtitle (subject to some restrictions; see manpage)
-#Ctrl+LEFT   no-osd sub-seek -1
-#Ctrl+RIGHT  no-osd sub-seek  1
-# Adjust timing to previous/next subtitle
-#Ctrl+Shift+LEFT sub-step -1
-#Ctrl+Shift+RIGHT sub-step 1
-# Move video rectangle
-#Alt+left  add video-pan-x  0.1
-#Alt+right add video-pan-x -0.1
-#Alt+up    add video-pan-y  0.1
-#Alt+down  add video-pan-y -0.1
-# Zoom/unzoom video
-#Alt++     add video-zoom   0.1
-#Alt+-     add video-zoom  -0.1
-# Reset video zoom/pan settings
-#Alt+BS set video-zoom 0 ; set video-pan-x 0 ; set video-pan-y 0
-#PGUP add chapter 1                     # skip to next chapter
-#PGDWN add chapter -1                   # skip to previous chapter
-#Shift+PGUP seek 600
-#Shift+PGDWN seek -600
-#[ multiply speed 1/1.1                 # scale playback speed
-#] multiply speed 1.1
-#{ multiply speed 0.5
-#} multiply speed 2.0
-#BS set speed 1.0                       # reset speed to normal
-#Shift+BS revert-seek                   # undo previous (or marked) seek
-#Shift+Ctrl+BS revert-seek mark         # mark position for revert-seek
-#q quit
-#Q quit-watch-later
-#q {encode} quit 4
-#ESC set fullscreen no
-#ESC {encode} quit 4
+#Shift+RIGHT no-osd seek  1 exact       # seek exactly 1 second forward
+#Shift+LEFT  no-osd seek -1 exact       # seek exactly 1 second backward
+#Shift+UP    no-osd seek  5 exact       # seek exactly 5 seconds forward
+#Shift+DOWN  no-osd seek -5 exact       # seek exactly 5 seconds backward
+#Ctrl+LEFT   no-osd sub-seek -1         # skip to the previous subtitle
+#Ctrl+RIGHT  no-osd sub-seek  1         # skip to the next subtitle
+#Ctrl+Shift+LEFT sub-step -1            # change subtitle timing such that the previous subtitle is displayed
+#Ctrl+Shift+RIGHT sub-step 1            # change subtitle timing such that the next subtitle is displayed
+#Alt+left  add video-pan-x  0.1         # move the video left
```

Comment:
technically the video is moving right when pressing 'alt+left'. The binding still makes if you think of it as the camera/viewpoint moving left